.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,96 @@
+name: New Issue
+description: Create a new issue
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
+         server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
+      3. Please take a moment to check that your issue doesn't already exist.
+      4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
+      5. Please give all relevant information below for bug reports, because
+         incomplete details will be handled as an invalid report.
+- type: input
+  id: gitea-ver
+  attributes:
+    label: Gitea Version
+    description: Gitea version (or commit reference) your instance is running
+    placeholder: v1.14.3
+  validations:
+    required: true
+- type: input
+  id: git-ver
+  attributes:
+    label: Git Version
+    description: The version of git running on the server's systemm
+  validations:
+    required: true
+- type: input
+  id: os-ver
+  attributes:
+    label: Operating System
+    description: The operating system you are using to run Gitea
+  validations:
+    required: true
+- type: textarea
+  id: run-info
+  attributes:
+    label: How are you running Gitea?
+    description: |
+      Please include information on whether you built gitea yourself, used one of our downloads or are using some other package
+      Please also tell us how you are running gitea, e.g. if it is being run from docker, a command-line, systemd etc.
+      If you are using a package or systemd tell us what distribution you are using
+  validations:
+    required: true
```

Comment:
Also not always required. What should I say about try.gitea.io?

Reply:
As far as I am aware, try.gitea.io is a testing environment only.  There is a section below regarding using try.gitea.io to test things that you find on your own installations.
This is more checking wheter they are using https://gitea.com, another hosted environment or their own self hosted variation.
I can remove the required flag from this input, however I bleieve that it would be valuable information for users to provide in terms of where they are using Gitea.

Reply:
Yeah, I know.
But the comfort of try.gitea.com is exactly that the latest features can already be tested. And sometimes, there are bugs in these new features. Then, you can refer to try.gitea.com for these bugs.
That's what I mean by that.

Reply:
To the best of my knowledge, try.gitea.io is a testing environment only and should only be used for reproducing bugs and trying out gitea.
I think that this would be valuable information for contributors and collaborators to have, so instead of disabling the requirement I might add something that prompts the person to write if they are using try.gitea.io .
Thoughts?

Reply:
Okay, sounds fine.
(Since I am most likely affected the most by this - I'd say that I am the person with the highest count of issues opened from testing on try.gitea.com) that sounds reasonable.
Yeah, I can support that.